client/src/components/LibraryNavigator/LibraryModel.ts

```diff
@@ -25,7 +25,7 @@ class LibraryModel {
 
   public async connect(): Promise<void> {
     const session = getSession();
-    session.onLogFn = LogChannelFn;
+    session.onExecutionLogFn = appendSessionLogFn;
```

Comment:
why not
```suggestion
    session.onSessionLogFn = appendSessionLogFn;
```

Reply:
fixed in https://github.com/sassoftware/vscode-sas-extension/pull/713/commits/6d6d1e0dd3d0938d427f339603b093b34cf7d2f0